src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -11,9 +11,17 @@ Released 2023-Sep-05
 Released 2023-Aug-21
 
 * **Breaking change**: Excluded attributes corresponding to `LogRecord.EventId`,
-`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. This
-is done as the semantic conventions for these attributes are not yet stable.
-([#4781](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4781))
+`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data.These
```

Comment:
Update

> CategoryName - same as above. Spec does talk about this (and recommend storing as instrumentation library IIRC). So could be just matter of proper mapping instead of creating any conventions to store as attributes.

`InstrumentationScope` but not mentioning it here for now, could be added to discussions on the issue